tests/e2e/specs/customizer/header-builder/logo/site-logo-width.test.js

```diff
@@ -0,0 +1,45 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize, uploadImage, setLogo } from '../../../../utils/customize';
+import { setBrowserViewport } from '../../../../utils/set-browser-viewport';
+
+describe( 'Site logo settings in the customizer', () => {
+	it( 'site logo width should apply correctly', async () => {
+		const fileDetails = {
+			fileName: 'Logo-Image.jpg',
+			fileURL: 'https://cdn.pixabay.com/photo/2019/10/15/03/16/black-and-white-4550471_1280.jpg',
```

Comment:
```suggestion
			fileURL: 'https://cdn.pixabay.com/photo/2019/10/15/03/16/black-and-white-4550471_1280.jpg',
```
@Nikschavan Can we upload images somewhere on our hosted websites instead of getting them from third-party websites?

Reply:
We can keep the image uploaded in the repo and make it available in the wp-env and then download it.

This image also needs to be maintained and should not be deleted for some reason, if we keep this image somewhere else we might forget it and accidentally delete it in the future.

For now though I would say we can merge this as it is or use an external example URLs if available and add a task to do above suggestion in a separate PR.

Reply:
@Nikschavan Got it! Added new task for the same - https://wp-astra.atlassian.net/browse/QAE-263